features/ua-client-hints.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-10-21
-name: User-Agent Client Hints
-description: TODO
+name: User agent client hints
+description: The `Sec-CH-UA` HTTP request header and the `navigator.userAgentData` API expose browser and platform information used to vary responses and application logic. User agent client hints are similar to user agent strings but more privacy-preserving and more robust.
```

Optional: I don't know what "robust" means in this context, so I'm going to suggest a more specific detail:

```suggestion
description: The `Sec-CH-UA` HTTP request header and the `navigator.userAgentData` API expose browser and platform information used to vary responses and application logic. User agent client hints are similar to user agent strings but more privacy-preserving and safer to parse.
```